examples/deadline/All-In-AWS-Infrastructure-SEP/python/package/app.py

```diff
@@ -45,6 +45,7 @@ def main():
     sep_props = sep_stack.SEPStackProps(
         docker_recipes_stage_path=os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, 'stage'),
         worker_machine_image=MachineImage.generic_linux(config.deadline_client_linux_ami_map),
+        deadline_resource_tracker_exists=config.create_resource_tracker_role,
```

Comment:
this logic doesn't seem intuitive...

shouldn't it be:

```suggestion
        deadline_resource_tracker_exists=(not config.create_resource_tracker_role),
```

Reply:
Ah thanks for catching this. I forgot to update the named argument after changing the prop name. Updated

Reply:
I don't see `deadline_resource_tracker_exists` in the `SEPStackProps` at all. Should this be named `create_resource_tracker_role`?